forge-gui/res/cardsfolder/upcoming/zimones_hypothesis.txt

```diff
@@ -0,0 +1,7 @@
+Name:Zimone's Hypothesis
+ManaCost:3 U U
+Types:Instant
+A:SP$ PutCounter | Choices$ Creature | ChoiceTitle$ Choose a creature | CounterType$ P1P1 | CounterNum$ 1 | Optional$ True | SubAbility$ DBChooseEvenOdd | SpellDescription$ You may put a +1/+1 counter on a creature. Then choose odd or even. Return each creature with power of the chosen quality to its owner's hand. (Zero is even.)
+SVar:DBChooseEvenOdd:DB$ ChooseEvenOdd | Defined$ You | ListTitle$ Choose ODD or EVEN | SubAbility$ DBChangeZoneAll
+SVar:DBChangeZoneAll:DB$ ChangeZoneAll | ChangeType$ Creature.powerChosenEvenOdd | Origin$ Battlefield | Destination$ Hand
+Oracle:You may put a +1/+1 counter on a creature. Then choose odd or even. Return each creature with power of the chosen quality to its owner's hand. (Zero is even.)

```

Comment:
If this works, then Extinction Event should be probably rewritten.

Reply:
oops, most likely I commented too quick here
but you're lightning fast haha

on the other hand `cmcChosenEvenOdd` definitely exists

But I might not have time to support it for this too soonish